alea/models/blueice_extended_model.py

```diff
@@ -535,7 +548,15 @@ def _set_default_ptype(self):
         If no ptype is specified, set the default ptype "needs_reinit".
 
         """
-        allowed_ptypes = ["rate", "shape", "index", "efficiency", "livetime", "needs_reinit"]
+        allowed_ptypes = [
+            "rate",
+            "shape",
+            "index",
+            "efficiency",
+            "livetime",
+            "needs_reinit",
+            "mass",
```

Comment:
This has no meaning though (i.e. particular feature), right?

Reply:
what exactly are you referring to? It's actually just a reformatting for the code 
for the "mass" term, let me remove

Reply:
Alright then. Usually we use these type to mark parameters with special features and mass is not one of them, that's what i wanted to raise -- so basically suggesting to removing it here, yes.

Reply:
Removed